docs/schema.json

```diff
@@ -1157,11 +1155,13 @@
         },
         "selectedOutboundFlight": {
           "typeRef": "#/domains/FlightBooking/types/Flight",
-          "initial": true
+          "initial": true,
+          "fooBarBaz": ""
```

Comment:
naming: in some domains `selectedBla` doesn't mean it depends on `availableBla`, so we can't rely on `selected-available` pattern while building front-end